src/features/weather/weatherToday/weatherToday.tsx

```diff
@@ -0,0 +1,166 @@
+import { RootState } from 'app/store';
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  Filler,
+  Legend,
+  LineElement,
+  LinearScale,
+  PointElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import {  iconUrlFromcode, timeFormat } from 'components/format';
+import * as React from 'react';
+import { Line } from 'react-chartjs-2';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCoordinates } from '../coordinateSlice';
+import { fetchWeather } from '../weatherSlice';
+import {
+  Banner,
+  BannerContent,
+  BannerLeft,
+  BannerRight,
+  BannerTitle,
+  Chart,
+  City,
+  Icon,
+  Image,
+  ItemMonth,
+  Items,
+  Predict,
+  PredictItem,
+  Temp,
+  TitleChart,
+  Wrapper,
+} from './styles';
+import dayjs from 'dayjs';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend
+);
+
+const options = {
+  responsive: true,
+  plugins: {
+    legend: {
+      position: 'top' as const,
+    },
+    title: {
+      display: false,
+      text: 'Chart.js Line Chart',
+    },
+  },
+};
+
+
+export interface ITodayProps {}
+export default function Today(props: ITodayProps) {
+  const [isActive, setIsActive] = React.useState(0);
+  const [city, setCity] = React.useState('hanoi');
+  const dispatch = useDispatch<any>();
+  const weatherData = useSelector((state: RootState) => state.weather);
+  const coordinates = useSelector((state: RootState) => state.coordinate);
+  const Hourly = coordinates?.data?.hourly;
+  const Daily = coordinates?.data?.daily?.filter((daily, index) => index < 6);
```

Comment:
Biến không nên viết in hoa, nên là dailyList chẳng hạn
Lấy 7 ngày thì có thể dùng slice